fpx/src/api/client.rs

```diff
@@ -48,14 +48,19 @@ impl ApiClient {
         &self,
         method: Method,
         path: impl AsRef<str>,
+        body: Option<String>,
```

Comment:
This `body` should be a `body: Option<B>` and then we can add `B: serde::ser::Serialize`.